earth_enterprise/src/fusion/autoingest/sysman/khResourceProvider.cpp

```diff
@@ -136,7 +137,8 @@ khResourceProvider::Run(void)
   Systemrc systemrc;
   LoadSystemrc(systemrc);
   uint32 numCPUs = systemrc.maxjobs;
-
+  PERF_CONF_LOGGING( "rprovider_config_numcpus", "numcpus", numCPUs  );
+  notify(NFY_WARN, "khResourceProvider: systemrc.maxjobs =  %d",  numCPUs  );
   // start the SignalLoop thread - listens for SIGINT & SIGTERM
```

Comment:
Is this notify call staying in the merged code?

Reply:
Yes, that notify call actually writes to the logfile output in standard GEE output.